superset/assets/javascripts/explorev2/components/ChartContainer.jsx

```diff
@@ -66,8 +75,8 @@ class ChartContainer extends React.Component {
       || nextProps.height !== this.props.height);
   }
 
-  componentDidUpdate() {
-    this.renderVis();
+  componentDidUpdate(prevProps) {
```

Comment:
do we need both lifecycle methods componentWillReceiveProps and componentDidUpdate? Seems like it'd be simpler to do it all in one place.